sampleChangerApp/src/sampleChanger.h

```diff
@@ -2,6 +2,7 @@
 #define SAMPLECHANGER_H
  
 #include <string>
+#include <inttypes.h>
```

Comment:
i don't think you need this include, int types are part of the standard? i might be being dumb